Logstash/logstash-input-azureeventhub/lib/logstash/inputs/azureeventhub.rb

```diff
@@ -40,43 +40,76 @@ def register
   end # def register
 
   def get_pay_load(message)
-    return nil if not message
+    return nil, nil if not message
+    annotationMap = nil
+    body = nil
     message.getPayload().each do |section|
-      if section.java_kind_of? org::apache::qpid::amqp_1_0::type::messaging::Data or section.java_kind_of? org::apache::qpid::amqp_1_0::type::messaging::AmqpValue
-        return section.getValue().to_s.gsub("\\x5c", "\\")
+      if section.java_kind_of? org::apache::qpid::amqp_1_0::type::messaging::MessageAnnotations
+        annotationMap = section.getValue()
+      else
+        if section.java_kind_of? org::apache::qpid::amqp_1_0::type::messaging::Data or section.java_kind_of? org::apache::qpid::amqp_1_0::type::messaging::AmqpValue
+          body = section.getValue().to_s.gsub("\\x5c", "\\")
```

Comment:
Is it possible that the message contains multiple AmqpValues or one message will always have one value? Consider `body << section.getValue().to_s. .....` if multiple values is possible.

Reply:
no, there's only one AmqpValue